client/app/scripts/superdesk-ingest/module.js

```diff
@@ -59,6 +59,10 @@ define([
         search: {
             label: 'Search provider',
             templateUrl: 'scripts/superdesk-ingest/views/settings/searchConfig.html'
+        },
+        whatsapp: {
```

Comment:
If I understood correctly, he wants to change/override the 'providerTypes' value?